src/arc.ts

```diff
@@ -224,6 +224,7 @@ export async function initializeArc(provider?: any): Promise<boolean> {
     // get contract information from the subgraph
     if ((window as any).arc) {
       arc = (window as any).arc;
+      arc.web3 = new Web3(provider);
```

Comment:
The network can change.  Is this sufficient if you haven't also updated:

```
graphqlHttpProvider: arcSettings.graphqlHttpProvider ,
graphqlWsProvider: arcSettings.graphqlWsProvider,
```

Reply:
Perhaps I misunderstand, but no, the network should not ever change - the network is set on starting the app with an environment variable. E.g. if we are on main (or on rinkeby) we will always stay on main (or on rinkeby).

Reply:
Yeah I just remembered that we just recently coded the app to disallow using an "incorrect" network, so indeed the network effectively cannot change.